config/config.go

```diff
@@ -36,8 +36,18 @@ var (
 		"mocktikv": true,
 		"tikv":     true,
 	}
+	// CheckBeforeDrop enable to execute `admin check table` before `drop table`.
+	CheckBeforeDrop = false
+	// CheckBeforeDropLDFlag is a go build flag.
+	CheckBeforeDropLDFlag = "None"
 )
 
+func init() {
+	if CheckBeforeDropLDFlag == "1" {
+		CheckBeforeDrop = true
```

Comment:
use atomic to protect `CheckBeforeDrop`, or the data race test perhaps reports error.

Reply:
done.